pom.xml

```diff
@@ -74,8 +75,6 @@
   <properties>
     <packages.export>jakarta.inject.*</packages.export>
     <spec_version>2.0</spec_version>
-    <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Why do we delete this?

Reply:
What about other version of releases then? Does 9 means 9 and above?

Reply:
Sure, you can load classes compiled for 9 using JRE 11, 17 or any other newer version.